t_cam: [0, 0, 0]
cam_to_Rover_rot: [0,0,1,-1,0,0,0,-1,0]
camera_matrix: [236.77842459886676, 0.0, 180.5, 0.0, 236.77842459886676, 120.5, 0.0, 0.0, 1.0]